### ISSUE TITLE
Add support for multiple configurations in one yaml file.

### ISSUE THREAD

**Issue by prestonvasquez** [alpstable/gidari#131]
Currently configuration files have [this](https://github.com/alpine-hodler/gidari/blob/main/internal/transport/testdata/upsert/coinbase_pro.yml) shape where you can only define one API per file. We want to be able to define multiple API per configuration file, something like this:

```
operations: 
  - 
    url: x
    ...  
  -
    url: y
    ...
```

**Comment by heggrik** [comment]
I'd like to take a crack at this, tough it might take me a while to get familiar with the code.

**Comment by prestonvasquez** [comment]
@heggrik Absolutely, thank you! You're assigned.

**Comment by prestonvasquez** [comment]
Closed by #360
